service/src/main/java/gov/nasa/pds/api/registry/model/properties/PdsProperty.java

```diff
@@ -0,0 +1,54 @@
+package gov.nasa.pds.api.registry.model.properties;
+
+import gov.nasa.pds.api.registry.exceptions.UnsupportedSearchProperty;
+import gov.nasa.pds.api.registry.model.exceptions.UnhandledException;
```

Comment:
this new class sounds like a good idea. We need to keep in mind the migration of the other conversion that you mentioned in comments.

Reply:
There's a ticket (indirectly) open for that - https://github.com/NASA-PDS/registry-api/issues/528

I'll update that ticket with the additonal detail now.